deploy.rb

```diff
@@ -87,6 +91,8 @@ def comment_out_branch_prompt!
 
 auditor = Auditor.new
 
+puts "repos to #{ssh_check ? 'ssh_check' : 'deploy'}: #{repo_names.join(', ')}"
```

Comment:
this is noise to me, because each repo is printed below anyway, multiple times (installing gems, running ssh_check or running deploy ...